src/app/utils.py

```diff
@@ -518,26 +518,22 @@ def check_spdx_license(licenseText):
     spdxLicenseTexts = r.mget(spdxLicenseIds)
     licenseData = dict(list(zip(spdxLicenseIds, spdxLicenseTexts)))
     matches = get_close_matches(licenseText, licenseData)
-
     if not matches:
         matchedLicenseIds = None
         matchType = 'No match'
-    
-    elif 1.0 in list(matches.values()) or all(0.99 < score for score in list(matches.values())):
-        matchedLicenseIds = list(matches.keys())
-        matchType = 'Perfect match'
-    
     else:
-        for licenseID in matches:
-            listedLicense = getListedLicense(licenseID)
-            isTextStandard = checkTextStandardLicense(listedLicense, licenseText)
-            if not isTextStandard:
-                matchedLicenseIds = licenseID
-                matchType = 'Standard License match'
-            else:
-                matchedLicenseIds = max(matches, key=matches.get)
-                matchType = 'Close match'
```

Comment:
`Standard License match` were overwritten by `Close match` matches. But I assume that Standard License matches are better then close matches

Reply:
Correct